### PR TITLE
deploy: helm charts templates for single-node

### PR DESCRIPTION
**Pull request by dextercai**
deploy: helm charts templates for single-node

It is for testing purposes only and is not production-ready.

### CODE REVIEW

**Comment by dextercai** [comment]
I have read the CLA Document and I hereby sign the CLA below to sign it.

**Comment by dextercai** [comment]
I have read the CLA Document and I hereby sign the CLA

File: install/helm-single/readme.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+首次部署完毕后，不会自动创建用户，请手动在Backend的Pod中执行
+
+```
+hydrooj cli user create systemjudge@systemjudge.local root rootroot
+hydrooj cli user setSuperAdmin 2
+```
+
+Helm Chart示例中尚未完全适配多节点以及HA需求。主要体现在
+- Mongo的单节点部署
+- 为了理解和调试便利，后端容器`/data/file`和`/root/.hydro`，Mongo容器`/data/db`，评测机容器``使用了HostPath。
```
Comment (undefined-moe) [inline install/helm-single/readme.md]: Incorrect formatting

File: install/helm-single/templates/backend.deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+              name: config-json-volume
+          ports:
+            - name: server-port
+              containerPort: 80
+          command: ["/bin/sh", "-c"]
+          args:
+            - |
+              cp /root/.hydro-config-map/addon.json /root/.hydro/ &&
+              cp /root/.hydro-config-map/config.json /root/.hydro/ &&
+              pm2-runtime start hydrooj
```
Comment (undefined-moe) [inline install/helm-single/templates/backend.deployment.yaml]: If you are using a container to manage all the processes, it's recommended to drop pm2 and simply start with `hydrooj`.

NOTE: pm2 supports forking multiple app processes and allow them listening on the same port to provide load balancing functionality, is there an alternative option for k8s?

**Review by undefined-moe — APPROVED** [review]

**Pull request merged**